app/src/hooks/api/useObservationApi.test.ts

```diff
@@ -41,4 +42,47 @@ describe('useObservationApi', () => {
     expect(result.blocks[0].id).toEqual(1);
     expect(result.blocks[1].id).toEqual(2);
   });
+
+  it('createBlockObservation as expected', async () => {
+    const observation = {
+      metaData: {
+        block_name: 1,
+        block_size: 1,
+        strata: 'c',
+        date: 'd',
+        start_time: 'e',
+        end_time: 'f',
+        pilot_name: 'g',
+        navigator: 'h',
+        rear_left_observer: 'i',
+        rear_right_observer: 'j',
+        visibility: 'k',
+        light: 'l',
+        cloud_cover: 1,
+        temperature: 1,
+        precipitation: 'o',
+        wind_speed: 1,
+        snow_cover: 10,
+        snow_depth: 10,
+        days_since_snowfall: 7,
+        weather_description: 't',
+        description_of_habitat: 'u',
+        aircraft_company: 'v',
+        aircraft_type: 'w',
+        aircraft_registration_number: 123,
+        aircraft_gps_model: 'y',
+        aircraft_gps_datum: 'z',
+        aircraft_gps_readout: 'zz'
+      },
+      tableData: { data: [['1', '2']] }
```

Comment:
instead of redefining this here, can just use what is in `observation-helpers.ts`
will save like 30 lines of code